cadence_language_and_execution_working_group/meetings/2024-02-20.md

```diff
@@ -0,0 +1,25 @@
+# Meeting 1
+
+## Date: Feb 20th, 2024, Tuesday, 9-10 AM PST
+
+## How to add an agenda topic?
+Any attendee can add an agenda topic by creating a pull request on this file and adding more topics in the agenda below.
+
+## Agenda
+
+Agenda:
```

Comment:
@j1010001 I added the agenda from the meeting invite here since I want a sharable link to the agenda.